docs/source/code/spatialstats.py

```diff
@@ -0,0 +1,40 @@
+import xdem
+import geoutils as gu
+import numpy as np
+
+# load diff and mask
+xdem.examples.download_longyearbyen_examples(overwrite=False)
+
+reference_raster = gu.georaster.Raster(xdem.examples.FILEPATHS["longyearbyen_ref_dem"])
+to_be_aligned_raster = gu.georaster.Raster(xdem.examples.FILEPATHS["longyearbyen_tba_dem"])
+glacier_mask = gu.geovector.Vector(xdem.examples.FILEPATHS["longyearbyen_glacier_outlines"])
+inlier_mask = ~glacier_mask.create_mask(reference_raster)
+
+nuth_kaab = xdem.coreg.NuthKaab()
```

Comment:
I think we should have a test case of two aligned DEMs, to avoid running the Nuth & Kaab each time...

Reply:
+1
@erikmannerfelt What do you think is the best approach here? Should we run the N&K once in `xdem/example.py` and store it very much like the raw data, maybe?

Reply:
Hmm. That's good for performance, but I don't see a good way to do it. Ideas?

Reply:
If we encapsulate the `download_longyearbyen_example` function into a `prepare_longyearbyen_example` functions which runs the download + runs N&K and writes to disk, would that work?

We would need to import xdem into `examples.py` which... shouldn't raise a circular import? (I'm not sure really)

Reply:
My idea was to just align it once and add it to xdem-data.

Reply:
Otherwise, there could be an examples function that coregisters and saves the DEM. Then all functions can just "get-or-generate" the DEM (so it only has to be generated once).

Perhaps this fits as a PR, as a large part of the documentation and test suite will need changing?